### PR TITLE
Adding src/de/2025-01/04/teacher-questions.md and previous

### PR DESCRIPTION
**Pull request by toak**
for adult lesson de/2024-04
from wiki.sabbatschule.at

### Pull Request check-list
_Please make sure to review and check all of these items:_
- [x] Have you checked for semicolons in the titles?

### CODE REVIEW

File: src/de/2024-04/03/teacher-questions.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ date:   18/10/2024
 
 -   Was in diesem Abschnitt könnt ihr logisch nicht nachvollziehen?
 
+```{=html}
```
Comment (VitalikL) [inline src/de/2024-04/03/teacher-questions.md:12]: @toak Unfortunately we can no longer accept improperly formatted files, please clean it up from using non-markdown format.

File: src/de/2024-04/03/teacher-questions.md [hunk 2]
```diff
@@ -86,8 +97,10 @@ date:   18/10/2024
...
 -   *Was berührt euch ganz persönlich an Johannes 1,1-18?*
 
+\-\-\--[^\[1\]^](4/2024_Lektion_03#_ftnref1 "wikilink") Bruce Milne,
 *The message of John: here is your king!: with study guide*, The Bible
 Speaks Today (Leicester, England; Downers Grove, IL: InterVarsity Press,
 1993), 4
 
-Weitere Fragen oder Ideen? - du bist herzlich zur Mitarbeit eingeladen: https://wiki.sabbatschule.at
+Weitere Fragen oder Ideen? - du bist herzlich zur Mitarbeit eingeladen:
+<https://wiki.sabbatschule.at>
```
Comment (VitalikL) [inline src/de/2024-04/03/teacher-questions.md:106]: Links should also be properly formatted

**Review by VitalikL — CHANGES_REQUESTED** [review]
@toak Unfortunately we can no longer accept improperly formatted files, please clean it up from using non-markdown format. The changes requested look like they are `auto-generated`

**Comment by VitalikL** [comment]
@toak I spent some time to clean up almost all previous `teacher-questions` - you can take a look, but even then the changes I made are not ideal. Ideally we would like clean Markdown format, because we are preparing big upgrade to our API and the app to avoid HTML completely.

**Comment by toak** [comment]
Thanks for the heads-up, I'll update the routine to remove HTML

**Comment by toak** [comment]
The source of the content is wiki.sabbatschule.at
I have changed the converter now to be markdown_strict without HTML: (new commits)
pandoc -f mediawiki -t markdown_strict-raw_html

Let me know what else you want to have improved @VitalikL

File: src/de/2024-04/12/teacher-questions.md [hunk 3]
```diff
@@ -11,71 +11,72 @@ date:   20/12/2024
...
-    -   \...
+    -   ...
+    -   ...
+    -   ...
 
-## MITTWOCH -- Das leere Grab
+## MITTWOCH – Das leere Grab
 
-\'\'\'Joh 20,1-10'''
+'''Joh 20,1-10‘‘‘
```
Comment (VitalikL) [inline src/de/2024-04/12/teacher-questions.md:57]: @toak This looks to me like a misformatted markdown. 
1. What is that supposed to be rendered at?
2. The closing three symbols are different that opening

File: src/de/2024-04/03/teacher-questions.md [hunk 4]
```diff
@@ -86,8 +94,10 @@ date:   18/10/2024
...
-Speaks Today (Leicester, England; Downers Grove, IL: InterVarsity Press,
-1993), 4
+----[^(\[1\])](4/2024_Lektion_03#_ftnref1 "wikilink") Bruce Milne, *The
+message of John: here is your king!: with study guide*, The Bible Speaks
+Today (Leicester, England; Downers Grove, IL: InterVarsity Press, 1993),
+4
 
-Weitere Fragen oder Ideen? - du bist herzlich zur Mitarbeit eingeladen: https://wiki.sabbatschule.at
+Weitere Fragen oder Ideen? - du bist herzlich zur Mitarbeit eingeladen:
+<https://wiki.sabbatschule.at>
```
Comment (VitalikL) [inline src/de/2024-04/03/teacher-questions.md:103]: @toak Can you please remove surrounding `<` and `>`

File: src/de/2024-04/06/teacher-questions.md [hunk 5]
```diff
@@ -3,15 +3,15 @@ title:  Gesprächsfragen SabbatschulWiki
 date:   08/11/2024
 ---
 
-#### Startfrage(n)
+#### **Startfrage(n)**
```
Comment (VitalikL) [inline src/de/2024-04/06/teacher-questions.md:6]: @toak There is no point in bolding the headings, they are always rendered with bold font

File: src/de/2024-04/12/teacher-questions.md [hunk 6]
```diff
@@ -11,71 +11,72 @@ date:   20/12/2024
...
+## SONNTAG – Was ist Wahrheit?
 
-**Joh 18,33-38**
+'''Joh 18,33-38““
 
 -   Fragen zum Text:
-    -   \...
-    -   \...
-    -   \...
+    -   ...
```
Comment (VitalikL) [inline src/de/2024-04/12/teacher-questions.md:21]: @toak What are three dots suppose to represent?

File: src/de/2024-04/03/teacher-questions.md [hunk 7]
```diff
@@ -9,57 +9,65 @@ date:   18/10/2024
 
 -   Was in diesem Abschnitt könnt ihr logisch nicht nachvollziehen?
 
+&nbsp;
```
Comment (VitalikL) [inline src/de/2024-04/03/teacher-questions.md:12]: @toak Not sure if we need non breaking space here

**Review by VitalikL — CHANGES_REQUESTED** [review]

**Comment by toak** [comment]
@VitalikL I'll fix the issues with the converter in the next days

**Comment by toak** [comment]
@VitalikL I've added some additional rules to the conversation process, so the current files should be clean. If you notice some other issues, please comment on the PR, so I can fix the converter.

File: src/de/2024-04/12/teacher-questions.md (hunk 3 above)
Comment (toak) [reply src/de/2024-04/12/teacher-questions.md]: that was an issue in the data source, it's fixed now

File: src/de/2024-04/03/teacher-questions.md (hunk 4 above)
Comment (toak) [reply src/de/2024-04/03/teacher-questions.md]: will always be removed

File: src/de/2024-04/06/teacher-questions.md (hunk 5 above)
Comment (toak) [reply src/de/2024-04/06/teacher-questions.md]: will always be removed

File: src/de/2024-04/12/teacher-questions.md (hunk 6 above)
Comment (toak) [reply src/de/2024-04/12/teacher-questions.md]: that is when the editor still need to put questions, is always removed in the transformation

**Comment by toak** [comment]
@VitalikL everything should now be ready for the pull

**Comment by toak** [comment]
@VitalikL anything you want me to change before you can pull it? I checked again and all should be fine...

**Comment by toak** [comment]
@VitalikL do you want me to change anything before you can merge? I regularly keep things up to date to avoid merge requests
I already got the questions from users when the questions will be available.

**Pull request closed without merging**